components/server/src/redis/mutex.ts

```diff
@@ -42,21 +41,6 @@ export class RedisMutex {
         duration: number,
         routine: (signal: RedlockAbortSignal) => Promise<T>,
     ): Promise<T> {
-        try {
-            return this.client().using(resources, duration, routine);
-        } catch (err) {
-            if (err instanceof ExecutionError) {
-                // Workaround for https://github.com/mike-marcacci/node-redlock/issues/168 and https://github.com/mike-marcacci/node-redlock/issues/169
```

Comment:
I don't think we faced the same problems described in these issues. Also Redlock throws ExecutionError by design. So this is expected.